### DISCUSSION TITLE
pygobject use

### DISCUSSION THREAD

**Discussion by immolo** [lxgr-linux/pokete#210] [Q&A] [answered]
I was trying to figure out what this game needs pygobject for to list as a dependency as the game seems to work without it and couldn't see it in the source (Although my coding sucks so that means nothing).

Could you enlightenment me please?

**Comment by lxgr-linux** [comment, marked as answer]
I mentioned this in some other places, but I will say it again more publicly.
PyGobject isn't a dependency of Pokete it self, but rather of playsound, because playsound behaves differently when PyGobject is installed or not. For example when it's not present, the thread in which playsound runs and plays music will run for ever and becomes unstoppable. That will result in all music playing at once and will make a lot of noise. This does not happen when PyGobject is present. I would also rather not have it as a dependency.

**Reply by immolo** [reply]
Thanks for the answer, at least I know I can build it without the Cairo dependency to save on space now.
